toolkit/offchain/src/scripts_data.rs

```diff
@@ -242,13 +242,17 @@ pub(crate) fn reserve_scripts(
 // Returns the simplest MultiSig policy configuration plutus data:
 // there is one required authority and it is the governance authority from sidechain params.
 pub(crate) fn multisig_governance_policy_configuration(
-	governance_authority: MainchainKeyHash,
+	governance_authority: Vec<MainchainKeyHash>,
```

Comment:
same here, you do not consume governance_authority, so take it by a reference or via a slice